lint/src/test/java/org/simple/clinic/lint/TextViewTextSizeDetectorTest.kt

```diff
@@ -0,0 +1,36 @@
+package org.simple.clinic.lint
+
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Issue
+
+@Suppress("UnstableApiUsage")
+class TextViewTextSizeDetectorTest : LintDetectorTest() {
+
+  fun `test having text size in text view should throw error`() {
+    lint()
+        .files(
+            xml("res/layout/text_view.xml", """
+              <TextView xmlns:android="http://schemas.android.com/apk/res/android"
+                  android:layout_width="wrap_content"
+                  android:layout_height="wrap_content"
+                  android:textSize="18sp"/>
+            """.trimIndent())
+        )
+        .run()
```

Comment:
I'm not able to run these tests it needs some android sdk. Can you guide me on how to run them?